src/components/views/right_panel/UserInfo.tsx

```diff
@@ -369,11 +369,14 @@ const UserOptionsSection: React.FC<{
                 });
             };
 
-            readReceiptButton = (
-                <AccessibleButton onClick={onReadReceiptButton} className="mx_UserInfo_field">
-                    { _t('Jump to read receipt') }
-                </AccessibleButton>
-            );
+            const room = cli.getRoom(member.roomId);
+            if (room && room.getEventReadUpTo(member.userId)) {
```

Comment:
This is my new favourite operator:
```suggestion
            if (room?.getEventReadUpTo(member.userId)) {
```